product_docs/docs/ocl_connector/15/04_open_client_library/03_compiling_and_linking_a_program.mdx

```diff
@@ -7,44 +7,35 @@ title: "Compiling and linking a program"
 
 The EDB Open Client Library allows applications written using the Oracle Call Interface API to connect to and access an EDB database with minimal changes to the C source code. The EDB Open Client Library files are named:
 
- On Linux:
+- On Linux: `libedboci.so`
 
- `libedboci.so`
-
- On Windows:
-
- `edboci.dll`
+- On Windows: `edboci.dll`
 
 The files are installed in the `oci/lib` subdirectory.
 
 ## Compiling and linking a sample program
 
 This example compiles and links the sample program `edb_demo.c` in a Linux environment. The `edb_demo.c` file is located in the `oci/samples` subdirectory.
 
-1.  Set the `ORACLE_HOME` and `EDB_HOME` environment variables.
-
-2.  Set `ORACLE_HOME` to the complete pathname of the Oracle home directory.
-
-    For example:
+1.  Set `ORACLE_HOME` to the complete pathname of the Oracle home directory, for example:
 
```

Comment:
@nidhibhammar This is where I deleted that step. I don't think it was saying anything that isn't said in the next two steps?